src/Analysis/GWASUIMockup/GWASUI.jsx

```diff
@@ -0,0 +1,483 @@
+import React, { useState, useCallback, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Steps, Button, Space, Table, Input, Form, Result, Alert, Popconfirm, InputNumber, Select,
+} from 'antd';
+import { userHasMethodForServiceOnResource } from '../../authMappingUtils';
+import GWASUIJobStatusList from './GWASUIJobStatusList';
+import './GWASUI.css';
+import { getAllSources, getCohortJsonByName } from '../../cohortMiddleware';
+
+const { Step } = Steps;
+
+const steps = [
+  {
+    title: 'Step 1',
+    description: 'Select a cohort for GWAS',
+  },
+  {
+    title: 'Step 2',
+    description: 'Select harmonized variables for phenotypes and covariates',
+  },
+  {
+    title: 'Step 3',
+    description: 'Select which variable is your phenotype and remove unwanted covariates',
+  },
+  {
+    title: 'Step 4',
+    description: 'Set workflow parameters',
+  },
+  {
+    title: 'Step 5',
+    description: 'Submit GWAS job',
+  },
+];
+
+const GWASUI = (props) => {
+  const [form] = Form.useForm();
+  const [current, setCurrent] = useState(0);
+  const [selectedCohort, setSelectedCohort] = useState(undefined);
+  const [selectedConcepts, setSelectedConcepts] = useState([]);
+  const [numOfPC, setNumOfPC] = useState(3);
+  const [selectedPhenotype, setSelectedPhenotype] = useState(undefined);
+  const [selectedCovariates, setSelectedCovariates] = useState([]);
+  const [jobName, setJobName] = useState(undefined);
+  const [showJobSubmissionResult, setShowJobSubmissionResult] = useState(false);
+  const [jobSubmittedRunID, setJobSubmittedRunID] = useState(undefined);
+  const [marinerJobStatus, setMarinerJobStatus] = useState([
```

Comment:
why put these dashboard pages as default values for `marinerJobStatus` if this component is no longer a mockup?

Reply:
engine solution not complete yet, will re-utilize the state variables in a future iteration